src/garage/sampler/default_worker.py

```diff
@@ -94,7 +94,7 @@ def update_env(self, env_update):
     def start_rollout(self):
         """Begin a new rollout."""
         self._path_length = 0
-        self._prev_obs = self.env.reset()
+        self._prev_obs, _ = self.env.reset()
```

Comment:
are you not going to transmit the `episode_infos`? I think you could just add fields to `TrajectoryBatch` and `TimeStep`/`TimeStepBatch` accordingly

Reply:
Please file an issue for this